lib/components/Modal.js

```diff
@@ -94,7 +94,7 @@ var Modal = React.createClass({
         this.portal.closeWithTimeout();
       }
 
-      setTimeout(this.removePortal.bind(this), closesAt - now);
+      setTimeout(function() { this.removePortal(); }, closesAt - now);
```

Comment:
I think `removePortal` won't be available on `this` in `setTimeout` which will have `this` context of `window`? If you don't use arrow function, it won't scope capture.